eduNEXT/atlas-ansible-utils
# Jhony/elasticsearch role

# Description

This PR adds the necessary playbooks, roles, and templates to use Elasticsearch 7.10.
## How to test it?
To test it, you should use [Ansible](https://github.com/eduNEXT/atlas-ansible-utils#atlas-ansible-utils) and use the command
```shell
ansible-playbook elasticsearch_7_10.yml -i ../your-inventory-repo/path/to/hosts.ini -v
